olm-catalog/minio-operator/2.0.0/minio-operator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,271 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: minio-operator.v2.0.0
+  namespace: placeholder
+  annotations:
+    categories: 'Storage,    Big Data, AI/Machine Learning'
+    certified: 'false'
+    description: ' MinIO Operator allows creating distributed MinIO Clusters and manage their lifecycle'
+    containerImage: 'minio/k8s-operator:2.0.0'
+    support: 'MinIO, Inc.'
+    capabilities: Full Lifecycle
+    repository: 'https://github.com/minio/minio-operator'
+    createdAt: 2019-10-21T00:00:00.000Z
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "v1",
+          "kind": "Secret",
+          "metadata": {
+            "name": "minio-creds-secret"
+          },
+          "data": {
+              "accesskey": "bWluaW8=",
+              "secretkey": "bWluaW8xMjM="
+          }
+        },
+        {
+          "apiVersion": "operator.min.io/v1beta1",
+          "kind": "MinIOInstance",
+          "metadata": {
+            "name": "minio"
+          },
+          "spec": {
+            "replicas": 4,
+            "credsSecret": {
+              "name": "minio-creds-secret"
+            },
+            "requestAutoCert": false
+          }
+        }
+      ]
+spec:
+  displayName: MinIO Operator
+  description: >
+    ## About the managed application
+
+    MinIO object storage platform enables building high performance data
+    infrastructure for machine learning, analytics and application data
+    workloads. MinIO is Open Source, Enterprise-Grade, Amazon S3 Compatible
+    Object Storage. Some of the key features are:
+
+    * SQL Select
+
+    * Encryption & WORM
+
+    * Multi-Site Federation
+
+    * Lambda Compute
+
+    * Integration with IAM Tools
+
+    * Erasure Code & Bitrot Protection
+
+    ## About this Operator
+
+    MinIO Operator provides the following features:
+
+
+    * Deploy: Launch MinIO Clusters with specific affinity, toleration and other
+    pre-defined settings
+
+    * Automatic TLS: Deploy TLS enabled MinIO clusters with builtin certificate
+    signing requests.
+
+    * Configure: Configure the fundamentals of MinIO like versions, persistence,
+    erasure code configuration, and replicas.
+
+  maturity: stable
+  version: 2.0.0
+  minKubeVersion: 1.14.0
+  keywords:
+    - Object Storage
+    - Private Cloud
+    - S3 API
+    - Cloud Storage
+  maintainers:
+    - name: MinIO Developers
+      email: dev@min.io
+  provider:
+    name: 'MinIO, Inc.'
+  labels: {}
+  selector:
+    matchLabels: {}
+  links:
```

Comment:
We can add links for 

```
MinIO Subnet?
```

Reply:
I think we only have the `pricing` page https://min.io/pricing is this one ok?

Reply:
Yeah works @dvaldivia

Reply:
done